src/ansys/fluent/core/launcher/load_machines.py

```diff
@@ -0,0 +1,468 @@
+"""A module that provides machine list construction for distributed parallel
+environments, including queueing systems.
+
+Currently supports UGE, LSF, PBS and SLURM by parsing the contents of
+the PE_HOSTFILE, LSB_MCPU_HOSTS, PBS_NODEFILE and SLURM_JOB_NODELIST
+variables, respectively.
+"""
+import csv
+import os
+import subprocess
+
+from ansys.fluent.core.launcher.machine_list import Machine, MachineList
+
+
```

Comment:
So... the return value of load_machines would be used to construct a `-cnf` string to pass to Fluent.

Reply:
@dnwillia-work should we put these two new files in a subdirectory such as scheduling or job_scheduling?

Reply:
Yeah sure, I can move them

Reply:
Either way, I will approve...

Many thanks for adding this, @dnwillia-work

Reply:
It's done.